core/src/main/java/nu/yona/server/properties/BatchServiceProperties.java

```diff
@@ -8,6 +8,8 @@
 {
 	private String serviceUrl = "http://localhost:8083";
 
+	private String activityAggregationJobCronExpression = "0 0 3 * * *";
```

Comment:
I see. They have seconds as an extra field. It's weird that they're not more explicit about this.